DigitalLearningSolutions.Data/DataServices/CourseDataService.cs

```diff
@@ -127,5 +129,31 @@ public int GetNumberOfActiveCoursesAtCentreForCategory(int centreId, int adminCa
                 new { centreId, adminCategoryId }
             );
         }
+
+        public IEnumerable<CourseStatistics> GetCourseStatisticsAtCentreForCategoryId(int centreId, int categoryId)
+        {
+            return connection.Query<CourseStatistics>(

```

Comment:
Note that this query is designed to be generic so that it can be reused in other centre course pages so it does not filter fully down to match this ticket's requirements perfectly. Additional filtering is performed in the CourseService.

Reply:
This is perfect! :)